jenkins/aragorn/Jenkinsfile

```diff
@@ -121,6 +121,8 @@ pipeline {
                         ]) {
                             sh '''
                             aws --region ${AWS_REGION} eks update-kubeconfig --name ${KUBERNETES_CLUSTER_NAME}
+                            helm delete $APP -n sri 
+                            kubectl delete pvc redis-data-aragorn-redis-master-0 -n sri 
```

Comment:
Will this delete the existing PVC when we do a new deployment? I'm fairly sure we don't want this.